js/index.js

```diff
@@ -1 +1,69 @@
 // Your code goes here
+const Newtitle = 
+document.querySelector("title");
```

Comment:
in your query selector, you need to add a css selector inside of it or else it will look for an element tag that doesn't exist, for example: 
```
<div class="title"> </div>

document.querySelector(".title")

pay attention to the dot next to title
```